terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/config.ts

```diff
@@ -98,6 +100,8 @@ export class Config {
     this.scaleConfigRepo = process.env.SCALE_CONFIG_REPO || 'test-infra';
     this.scaleConfigRepoPath = process.env.SCALE_CONFIG_REPO_PATH || '.github/scale-config.yml';
     this.secretsManagerSecretsId = process.env.SECRETSMANAGER_SECRETS_ID;
+    this.sSMParamCleanupAgeDays = Number(process.env.SSM_PARAM_CLEANUP_AGE_DAYS || '7');
```

Comment:
Don't we occasionally have runners that live for longer than 7 days?  Perhaps we should be more conservative about this time range to account for worst case possibilities

Reply:
if you read the code, [when you asked where it is used](https://github.com/pytorch/test-infra/pull/5479#issuecomment-2246149918), you'll see that it should be deleted by the instance itself when it starts.

Those are side effects of instances failing to start for a reason or another